### PR TITLE
Create hexathon UI

### PR DESCRIPTION
**Pull request by gary-peng**
- Added a new modal on select events page to create hexathons.
- Visible to exec only.

image
image

### CODE REVIEW

File: src/components/selectEvent/HexathonModal.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                  <FormLabel>Start Date</FormLabel>
+                  <Input type="date" {...register("startDate")} />
+                </FormControl>
+                <FormControl isRequired>
+                  <FormLabel>End Date</FormLabel>
+                  <Input type="date" {...register("endDate")} />
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Email Header Image</FormLabel>
+                  <Input {...register("emailHeaderImage")} />
```
Comment (Roshan-D) [inline src/components/selectEvent/HexathonModal.tsx:58]: can you make this an image upload?

File: src/components/selectEvent/HexathonModal.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Cover Image</FormLabel>
+                  <Input {...register("coverImage")} />
```
Comment (Roshan-D) [inline src/components/selectEvent/HexathonModal.tsx:62]: can you make this an image upload?

File: src/components/selectEvent/HexathonModal.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import axios from "axios";
+import { apiUrl, Service } from "@hex-labs/core";
+
+const HexathonModal: React.FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  const { register, handleSubmit } = useForm();
+  const onSubmit = async (data: any) => {
+    await axios.post(
+      apiUrl(Service.HEXATHONS, `/hexathons`),
```
Comment (Roshan-D) [inline src/components/selectEvent/HexathonModal.tsx:13]: See [this comment](https://github.com/HackGT/api/issues/125#issuecomment-1439313671)

**Review by Roshan-D — CHANGES_REQUESTED** [review]

**Comment by gary-peng** [comment]
image
Changed the image fields to file upload.

**Review by Roshan-D — CHANGES_REQUESTED** [review]
- Could you now take everything in public/events and upload it to the bucket
- Have the actual hexathons in SelectEvent.tsx use the images stored in the cloud

**Review by Roshan-D — APPROVED** [review]
Dope!

**Pull request merged**